test/parallel/test-buffer-indexof.js

```diff
@@ -81,43 +81,54 @@ assert.equal(Buffer.from('ff').indexOf(Buffer.from('f'), 1, 'ucs2'), -1);
 // test hex encoding
 assert.equal(
     Buffer.from(b.toString('hex'), 'hex')
-    .indexOf('64', 0, 'hex'), 3);
+    .indexOf('64', 0, 'hex'),
```

Comment:
@targos asked:

> shouldn't multiline method calls be indented with 2 spaces ?

(Deleted my previous inaccurate response.)

This is [a bug in ESLint](https://github.com/eslint/eslint/issues/1801). As @nzakas [comments in that issue](https://github.com/eslint/eslint/issues/1801#issuecomment-134696514): 

> Yeah, I'd imagine this will be open for quite some time. It's pretty difficult to do correctly, but patches always welcome.

(That comment is from August.)